### PR TITLE
Support for non-glibc linux which use musl-libc

### PR DESCRIPTION
**Pull request by zemanlx**
By applying these changes we are able to compile cbmc code and run successfully all tests in alpine linux.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -33,18 +61,17 @@ matrix:
...
-    - os: osx
-      compiler: gcc
-      env: COMPILER=g++
-    - os: osx
-      compiler: clang
-      env: COMPILER=clang++
+
     - env: NAME="CPP-LINT"
-      script: scripts/travis_lint.sh || true
+      script: scripts/run_lint.sh master HEAD || true
```
Comment (tautschnig) [inline .travis.yml]: That appears to revert an earlier commit?!

**Comment by zemanlx** [comment]
Reverted back to travis_lint.sh

**Comment by peterschrammel** [comment]
@zemanlx, please squash 3790cab into d9fdbb37.

**Comment by kroening** [comment]
This needs a rebase, and would then be ready to go!

**Comment by zemanlx** [comment]
@kroening it has been rebased.

**Comment by zemanlx** [comment]
@tautschnig I have added more information why I have done these changes.

**Pull request merged**